app/routes/campaigns.js

```diff
@@ -115,9 +107,8 @@ router.get('/:id', (req, res) => {
     })
     // TODO: Refactor helpers.sendResponse to accept an error and know the codes based on custom
     // error class, to DRY.
```

Comment:
This is step 1 of 2 for completing this `TODO` -- next up would be using the `helpers.sendUnproccessibleEntityProcess` (introduced in #853) in a middleware function checking for required parameters.